lib/modules/manager/bazel-module/filters.ts

```diff
@@ -0,0 +1,7 @@
+import is from '@sindresorhus/is';
+
+export function isNotNullOrUndefined<T>(
```

Comment:
do we really need this wrapper? if yes it's probably a candidate for general utils.

Reply:
I have not come across another function that provides this filter capability and ensures that the resulting type is correct. Previously, I tried `is.plainObject()`. Unfortunately, the type for the resulting array is `(T | null)[]`. With this type guard, the type ends up being `T[]`.

Would you like me to add it to `lib/util/array`?

Reply:
I put up #22081 if we want to add it as a shared function.

Reply:
I have updated this PR to use the now merged #22081.